action.yml

```diff
@@ -62,15 +64,41 @@ outputs:
 runs:
   using: composite
   steps:
+  - name: Compute environment Python version
+      id: compute_env_python_version
+      shell: bash
```

Comment:
I'd prefer to use `shell: python` for readability.

Reply:
Does that work, since this happens before Python is set up?

Reply:
It does, but it depends on what Python is the default one in current VM.

Reply:
Rewritten in 6c6a502d2ba5999545f867e89a7e565710e77fe1.